drawing/pdf_draw_shapes.go

```diff
@@ -0,0 +1,247 @@
+/*
+ * Draw multiple shapes in a new PDF file.
+ *
+ * Run as: go run pdf_draw_shapes.go
+ */
+
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/unidoc/unipdf/v3/common/license"
+	"github.com/unidoc/unipdf/v3/contentstream/draw"
+	"github.com/unidoc/unipdf/v3/creator"
+)
+
+const licenseKey = `
+-----BEGIN UNIDOC LICENSE KEY-----
+Free trial license keys are available at: https://unidoc.io/
+-----END UNIDOC LICENSE KEY-----
+`
+
+func init() {
+	// Enable debug-level logging.
+	// unicommon.SetLogger(unicommon.NewConsoleLogger(unicommon.LogLevelDebug))
+
+	err := license.SetLicenseKey(licenseKey, `Company Name`)
+	if err != nil {
+		panic(err)
+	}
+}
+
+func main() {
+	// New creator with default properties (pagesize letter default).
+	c := creator.New()
+	c.NewPage()
+
+	err := drawLine(c)
+	if err != nil {
+		fmt.Printf("Error: %v\n", err)
+		os.Exit(1)
+	}
+
+	err = drawPolyLine(c)
+	if err != nil {
+		fmt.Printf("Error: %v\n", err)
+		os.Exit(1)
+	}
+
+	err = drawRectangle(c)
+	if err != nil {
+		fmt.Printf("Error: %v\n", err)
+		os.Exit(1)
+	}
+
+	err = drawEllipse(c)
+	if err != nil {
+		fmt.Printf("Error: %v\n", err)
+		os.Exit(1)
+	}
+
+	err = drawCurve(c)
+	if err != nil {
+		fmt.Printf("Error: %v\n", err)
+		os.Exit(1)
+	}
+
+	err = drawPolyBezierCurve(c)
+	if err != nil {
+		fmt.Printf("Error: %v\n", err)
+		os.Exit(1)
+	}
+
+	err = drawCurvePolygon(c)
+	if err != nil {
+		fmt.Printf("Error: %v\n", err)
+		os.Exit(1)
+	}
+
+	err = drawPolygon(c)
+	if err != nil {
+		fmt.Printf("Error: %v\n", err)
+		os.Exit(1)
+	}
+
+	c.WriteToFile("output.pdf")
```

Comment:
We should also check the error here. Also, the output should maybe be called `unipdf-draw-shapes.pdf`.
Could also add the output file to the repo so that the result is easy to see.